packages/cli/src/lib/cmds/play.ts

```diff
@@ -165,11 +163,12 @@ async function setupGooglePlay(args: PlayArgs): Promise<GooglePlay> {
     twaManifest.serviceAccountJsonFile = args.serviceAccountFile;
     await twaManifest.saveToFile(manifestFile);
   }
-  if (!validServiceAccountJsonFile(twaManifest.serviceAccountJsonFile)) {
-    throw new Error('enUS.messageServiceAccountJSONMissing');
+  if (!validServiceAccountJsonFile(twaManifest.serviceAccountJsonFile) ||
+    !twaManifest.serviceAccountJsonFile) {
```

Comment:
I think we can reverse the condition order here so that if `twaManifest.serviceAccountJsonFile` is `null` or `undefined` then `validServiceAccountJsonFile()` is not even called.
```suggestion
  if (!twaManifest.serviceAccountJsonFile ||
      !validServiceAccountJsonFile(twaManifest.serviceAccountJsonFile)) {
```